routes/projects.js

```diff
@@ -1,9 +1,47 @@
 var express = require('express');
 var router = express.Router();
+var request = require('request')
+
+// Load up the environment variables.
+require('dotenv').config()
 
 /* GET projects listing. */
-router.get('/', function(req, res, next) {
-  res.send('respond with a resource');
+router.get('/:user', function(req, res, next) {
+
+  // User info from keycloak
+  const user = req.params.user;
+
+  // Login with admin credentials
+  var username = process.env.USERNAME,
+  password = process.env.PASSWORD,
+  url = "https://" + username + ":" + password + "@bnc.brown.edu/xnat-dev";
```

Comment:
I think it would be nice to put the url maybe in and `env` so that it is configurable and we don't have to later change multiple places when we are using the production XNATs